web/src/app/schedules/useOverrideNotices.ts

```diff
@@ -19,13 +19,13 @@ export default function useOverrideNotices(
   scheduleID: string,
   value: SpanISO,
 ): Notice[] {
-  const { data, loading } = useQuery(scheduleQuery, {
+  const [{ data, fetching }] = useQuery({
+    query: scheduleQuery,
     variables: {
       id: scheduleID,
     },
-    pollInterval: 0,
   })
-  if (loading) {
+  if (fetching) {
```

Comment:
fetching will always be false with urql (since it'll use suspense), so we can skip that check